locales/en/sign-up.json

```
@@ -11,10 +15,11 @@
   "passwordPlaceholder": "Enter your password",
   "firstNamePlaceholder": "Enter your first name",
   "lastNamePlaceholder": "Enter your last name",
+  "inputValueRequired": "This field is required.",
   "inputImproperName": "Please use proper name.",
+  "usernameCharactersMinimum": "Username must contain at least 6 characters.",
   "usernameAlreadyUsed": "Your username has already been used. Please use a different one.",
   "emailIncorrectFormat": "Your email is incorrect. Please recheck the email format or use a different one.",
   "passwordCharactersMinimum": "Password must contain at least 8 characters.",
-  "passwordSimilarToUsername": "Password must be different from Username.",
-  "passwordRequiredFormat": "Password must contain at least 1 lowercase, 1 uppercase character and 1 number."
+  "passwordRequiredFormat": "Password must contain at least 1 uppercase, 1 special character and 1 number."
```

Looking for other website, It should write the error message not over than 2 lines. Is it better? @Raksani @LilBank 
```suggestion
  "passwordRequiredFormat": "Require 1 uppercase, 1 special character and 1 number."
```

---

After I saw the design, I agree.